esx_service/utils/vmdk_utils.py

```diff
@@ -64,6 +64,9 @@
 VMDK_RETRY_COUNT = 5
 VMDK_RETRY_SLEEP = 1
 
+# root for all the volumes
+VOLUME_ROOT = "/vmfs/volumes/"
```

Comment:
Not related to your change: the string "/vmfs/volumes" has been hardcoded everywhere in our code base. We should refactor those code to use this constant. Let's file an issue for tracking purpose.